moabb/paradigms/base.py

```diff
@@ -110,10 +105,9 @@ def process_raw(self, raw, dataset, return_epochs=False):
         if len(stim_channels) > 0:
             events = mne.find_events(raw, shortest_event=0, verbose=False)
         else:
-            ev_selected = {str(v): v for v in event_id.values()}
             try:
                 events, _ = mne.events_from_annotations(raw,
-                                                        event_id=ev_selected,
+                                                        event_id=event_id,
```

Comment:
This change is also introduced in #113 . Usually you'd want to have each PR be based off the current master. However, if we want to merge both PRs, we can just merge this one and close 113.